packages/core-tester-cli/src/commands/make/block.ts

```diff
@@ -0,0 +1,94 @@
+import { Address, configManager, models, NetworkName, slots } from "@arkecosystem/crypto";
+import { flags } from "@oclif/command";
+import delay from "delay";
+import { writeFileSync } from "fs";
+import { satoshiFlag } from "../../flags";
+import { copyToClipboard } from "../../utils";
+import { BaseCommand } from "../command";
+
+export class BlockCommand extends BaseCommand {
+    public static description: string = "send multiple transactions";
+
+    public static flags = {
+        ...BaseCommand.flagsConfig,
+        ...BaseCommand.flagsDebug,
+        number: flags.integer({
+            description: "number of blocks to generate",
+            default: 1,
+        }),
+        transactions: flags.integer({
+            description: "number of transactions to generate",
+            default: 0,
+        }),
+        transactionAmount: satoshiFlag({
+            description: "initial wallet token amount",
+            default: 2,
+        }),
+        transactionFee: satoshiFlag({
+            description: "transfer fee",
+            default: 0.1,
+        }),
+        passphrase: flags.string({
+            description: "passphrase of the forger",
+            default: "clay harbor enemy utility margin pretty hub comic piece aerobic umbrella acquire",
+        }),
+        write: flags.boolean({
+            description: "write the blocks to the disk",
+        }),
+    };
+
+    public async run(): Promise<models.IBlock[]> {
+        const { flags } = await this.make(BlockCommand);
+
+        configManager.setFromPreset(flags.network as NetworkName);
+
+        // We always start with the genesis block!
+        let previousBlock = configManager.get("genesisBlock");
+
+        const blocks: models.IBlock[] = [];
+
+        for (let i = 0; i < flags.number; i++) {
+            const milestone = configManager.getMilestone(previousBlock.height);
+            const delegate = new models.Delegate(flags.passphrase, configManager.get("pubKeyHash"));
+
+            const transactions = [];
+            for (let i = 0; i < flags.transactions; i++) {
+                transactions.push(
+                    this.signer.makeTransfer({
+                        ...flags,
+                        ...{
+                            amount: flags.transactionAmount + 10,
+                            transferFee: flags.transactionFee,
+                            recipient: Address.fromPassphrase(flags.passphrase),
+                        },
+                    }),
+                );
+
+                await delay(100);
```

Comment:
Why this?

Reply:
To avoid duplicate transactions caused by timestamps, anything below 100 causes duplicates.

Reply:
Then this is not enough as I still got duplicate txs - anything below 1000ms (1s) has a chance of ending up in the same second. But adding 1s delay here would make it too slow - if many transactions are to be generated.

What about using different amount or different sender or different recipient in order to avoid duplicates?

Reply:
Changing the recipient would probably be already enough, ideally we only use `genesisBlock.transactions.map(t => Address.fromPublicKey(t.senderPublicKey))` or `genesisBlock.transactions.map(t => t.recipientId)` so we can be sure the wallets exist.